frontend/src/features/workspace/components/CreateFormModal/CreateFormModalContent/CreateFormDetailsScreen.tsx

```diff
@@ -83,12 +86,25 @@ export const CreateFormDetailsScreen = (): JSX.Element => {
                 name="responseMode"
                 control={control}
                 render={({ field }) => (
-                  <FormResponseOptions {...field} showMrf={showMrf} />
+                  <WorkspaceRowsProvider>
+                    <FormResponseOptions
+                      {...field}
+                      showMrf={showMrf}
+                      hasSingpass={hasSingpass}
+                    />
+                  </WorkspaceRowsProvider>
                 )}
                 rules={{ required: 'Please select a form response mode' }}
               />
             </Skeleton>
             <FormErrorMessage>{errors.responseMode?.message}</FormErrorMessage>
+            {!hasSingpass || (
+              <InlineMessage mt="2rem">
+                {
+                  'The form you are trying to duplicate has Singpass authentication which is not supported for Multi-respondent forms.'
+                }
```

Comment:
just to check: are the braces required? I think can just put raw text as a child of `InlineMessage`

Reply:
oh yeah they're not needed, thanks for highlighting!